### DISCUSSION TITLE
Daily 17/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#191] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
El viernes hice la validación de los campos del formulario de registración y lo subí al repositorio. El sábado tuvimos nuestra reunión semanal sincrónica. También estuve repasando para el exàmen y por la tarde lo realicé. El resto del fin de semana no pude hacer mucho. Ayer estuve revisando la documentación que tenemos y lo que solicitan ahora, es bastante trabajo así que espero poder hacerlo. También estuve viendo las issues que nos faltan y hay muchas en las que no se ha avanzado.
Hoy voy a corregir algunos bugs que encontré en registración y login, tambien pasare el formato de registración a login y empezaré a asignarme tareas en las que no se haya avanzado.

**Comment by BraianTroncoso** [comment]
El sábado tuvimos nuestra reunión semanal sincrónica. También estuve repasando para el examen y lo realicé. Estuve todo el finde realizando mis tareas asignadas las cual pude finalizar con éxito.
Back para login refactorizado.
Una vez que ingresas te re direcciona al perfil con los datos de dicho usuario.
Agregué funcionalidad al boton de "cerrar sesion" ahora si cierra sesion y despues va al home.
Librerias nuevas, Request, implementacion de volley.
Hasheo de contraseñas para la DB( ahora la pass no va directo )
Testing sobre el codigo, lo probé todo ok.
También agg script para db.( asi todos tenemos el mismo archivo con los mismos datos si se quiere )
Clases, interfaces, paquetes, muchos cambios para el back(adaptando al ya existente)
Falta corregir -> Tengo que modificar una parte del registro para que me tomé la clave para hashear( se puede desde postman ) tengo que adaptar esa parte.
Mejorar el ruteo del controller, todo funciona pero podria quedar mas limpio el código, tambien voy a implementar la lib Lombok para un condigo mas legible.
Bug en activity de crear perfil, haciendo testing me encontré que cuando haces "click" en cualquier parte de la pantalla que no sea un boton o una caja te saca de la app o te da error.
Hoy empezaré a asignarme tareas en las que no se haya avanzado y solucionar algunos bugs.

**Comment by MarielaGomez** [comment]
Durante el fin de semana no he podido avanzar ya que por dificultades personales no he estado presente, logré realizar el cuestionario obligatorio el día Lunes, durante ésta semana avanzaré y finalizaré sobre las tareas que tengo asignadas

**Comment by yan0106** [comment]
El fin de semana estuve trabajando en la tarea de darle funcionalidad a subir foto de perfil, tratando de construir el código del lado del backend. Tuve inconvenientes con la importación de librerías. En parte pude solucionarlo. 
También ayudé a mis compañeros a levantar el servidor y a cambiar la contraseña de MySQL para conectar la base de datos. Realicé el cuestionario integrador. Hoy tuve clases.

**Comment by marceloedelgado** [comment]
Durante el fin de semana estuve estudiando para la Evidencia 4 de aprendizaje y trabajando en mis tareas de crear la vista amoliada de la tarjeta.
Logre desarrollar la activity_tarjeta_amoliada pero no pude hacer que funcione al llamarla desde el botón "Consultar" de la activity_catalogo. Estuve estudiando cuál puede ser la solución

**Reply by Valentino399** [reply]
Ayer logré hacer el examen de la evidencia 4 estuve viendo como se podría crear para que el usuario suba una foto de perfil dentro de la aplicación y completando un poco el curso de Java
